iOS/src/toga_iOS/widgets/timeinput.py

```diff
@@ -0,0 +1,94 @@
+import datetime
+
+from rubicon.objc import SEL, CGSize
+from travertino.size import at_least
+
+from toga_iOS.libs import (
+    NSCalendar,
+    NSCalendarUnit,
+    NSDateComponents,
+    UIControlContentHorizontalAlignmentLeft,
+    UIControlEventValueChanged,
+    UIDatePickerMode,
+)
+
+from .base import Widget
+from .dateinput import TogaDatePicker
+
+
+def py_time(native_time):
+    components = NSCalendar.currentCalendar.components(
+        NSCalendarUnit.Hour | NSCalendarUnit.Minute | NSCalendarUnit.Second,
+        fromDate=native_time,
+    )
+    return datetime.time(components.hour, components.minute, components.second)
+
+
+def native_time(py_time):
+    components = NSDateComponents.alloc().init()
+    components.setHour(py_time.hour)
+    components.setMinute(py_time.minute)
```

Comment:
```suggestion
    components.setMinute(py_time.minute)
    components.setSecond(py_time.second)
```
Still need this for min and max values.